src/Mod/PartDesign/App/FeatureBoolean.cpp

```diff
@@ -160,7 +160,7 @@ App::DocumentObjectExecReturn *Boolean::execute()
     result = refineShapeIfActive(result);
 
     if (!isSingleSolidRuleSatisfied(result.getShape())) {
-        return new App::DocumentObjectExecReturn(QT_TRANSLATE_NOOP("Exception", "Result has multiple solids: that is not currently supported."));
+        return new App::DocumentObjectExecReturn(QT_TRANSLATE_NOOP("Exception", "Result has multiple solids: enable compounds in a Body."));

```

Comment:
```suggestion
        return new App::DocumentObjectExecReturn(QT_TRANSLATE_NOOP("Exception", "Result has multiple solids: enable 'Allow Compounds' in the body."));

```

maybe we can change it to the actual property?